cypress/fixtures/api/billing-v2/billing-v2-subscribed-all.json

```diff
@@ -204,16 +204,16 @@
         },
         {
             "key": "apps",
-            "name": "Apps",
-            "description": "Use apps to transform, filter, and modify your incoming data. (Export apps not included, see the Data pipelines addon for product analytics.)",
+            "name": "Transformations",
+            "description": "Use transformations to filter or modify your incoming data. (Destinations not included, see the Data pipelines addon for product analytics.)",
```

Comment:
Changing these in our fixtures doesn't actually make the billing changes. I've updated the descriptions and things here: https://github.com/PostHog/billing/pull/531